### PR TITLE
Fixes #10930 : Make widget names and titles consistent

### PR DESCRIPTION
**Pull request by bkearney**

### CODE REVIEW

File: app/services/dashboard/loader.rb [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
   module Loader
     # Default widgets that are displayed on the dashboard
-    DEFAULT_WIDGETS = [ {:template=>'status_widget',       :sizex=>8,:sizey=>1,:name=> N_('Status table')},
-                        {:template=>'status_chart_widget', :sizex=>4,:sizey=>1,:name=> N_('Status chart')},
-                        {:template=>'reports_widget',      :sizex=>6,:sizey=>1,:name=> N_('Report summary')},
-                        {:template=>'distribution_widget', :sizex=>6,:sizey=>1,:name=> N_('Distribution chart')}]
+    DEFAULT_WIDGETS = [ {:template=>'status_widget',       :sizex=>8,:sizey=>1,:name=> N_('Host Configuration Status')},
+                        {:template=>'status_chart_widget', :sizex=>4,:sizey=>1,:name=> N_('Host Configuration Chart')},
+                        {:template=>'reports_widget',      :sizex=>6,:sizey=>1,:name=> N_('Latest Events')},
+                        {:template=>'distribution_widget', :sizex=>6,:sizey=>1,:name=> N_('Distribution Chart')}]
```
Comment (domcleal) [inline app/services/dashboard/loader.rb]: Missing "Run" from the name, should be "Run Distribution" to match.

**Review by domcleal — CHANGES_REQUESTED** [review]
There are conflicts, please rebase the PR onto develop.

**Comment by bkearney** [comment]
@domcleal changed the title of the Distribution chart, rebased, and pushed a new verison.

**Comment by domcleal** [comment]
The integration tests appear to be failing, please check and update if necessary.

**Comment by bkearney** [comment]
@domcleal can you kick off the test again? My first push did not have a migration, and I am not sure if this teting caught it.

**Comment by domcleal** [comment]
Tests are recorded against commits, and the page shows it tested fef2dd0.  Still, if you insist: [test foreman]

**Comment by bkearney** [comment]
@domcleal pushed new version which fixes the test and incorporates your suggestion to make the tltle "Run Distribution Chart"

**Review by domcleal — APPROVED** [review]

**Comment by domcleal** [comment]
Merged, thanks @bkearney.

**Pull request merged**